torch/_tensor_docs.py

```diff
@@ -2749,6 +2749,7 @@ def add_docstr_all(method, docstr):
     "is_pinned",
     r"""
 Returns true if this tensor resides in pinned memory.
+By default, the device wrt pinned memory will be selected automatically.
```

Comment:
This can say that the default device is the current accelerator and link to https://pytorch.org/docs/main/torch.html#accelerators

Reply:
Updated